gravitee-apim-portal-webui-next/src/app/log-in/log-in.component.scss

```
@@ -80,6 +80,11 @@
     }
   }
 
+  &__registration {
+    margin-top: 20px;
+    text-align: center;
+  }
+
```

The `Sign up` link should have the same style as the `Forgot password`.

I would suggest extracting current `__forgot-password` style into a common selector:

```
%link-style {
  margin-top: 5px;
  color: #{theme.$primary-main-color};
  text-align: center;
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
}
```

and extending it for both links:

```suggestion
    &__forgot-password {
      @extend %link-style;
    }

    &__error {
      text-align: center;
    }
  }

  &__registration {
    margin-top: 20px;
    text-align: center;

    a {
      @extend %link-style;
    }
  }
```
WDYT about this?

---

replaced as suggested